### PR TITLE
add env distinction and update TLS info

### PR DESCRIPTION
**Pull request by purplem1lk**
# PR Description

This PR fixes the issue number: 

### Summary of my changes and explanation of my decisions

- Add visuals of determining EngineBlock vs WebContainer-based projects
- Reword TLS renewal cert info and update yaml template

### Self-check

Please check all that apply:

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my code or content update and edited it to the best of my abilities
- [ ] I have commented my code, particularly in hard-to-understand areas; my comments are concise

### Contributors list

Would you like your contribution to be celebrated? Please check all that apply:

- [ ] I would like to be featured on the future contributors list in the README. If so, please tell us:
  - what name you'd like to be shown there?
  - we usually link github profile pic -- is that ok? If not, provide another url: 
  - we usually link your github profile but if you have a portfolio page, provide a link:

- [ ] I would like to get a shoutout in the next monthly updates post. If so, please provide your twitter handle (or we will link to your GitHub profile).

⚡️ ⚡️ ⚡️  Thank you for contributing to StackBlitz ⚡️ ⚡️ ⚡️

### CODE REVIEW

File: docs/guides/user-guide/available-environments.md [hunk 1]
```diff
@@ -56,3 +56,22 @@ This environment has a terminal that supports various common commands you would
...
+:::tip Not seeing a terminal in your project?
+
+Currently, the terminal is only available in projects which use the WebContainers execution environment. There is no way to automatically convert from one execution environment to another.
+:::
+
+The simplest way to determine if you're using EngineBlock vs WebContainers is the presence of a terminal, as shown below. If you see a terminal, you're in a WebContainers-based project.
+
+![Screenshot of the terminal in a StackBlitz WebContainer project](./assets/stackblitz_terminal.png)
+
+You can also check in the project settings in the left pane of any StackBlitz project. When you click the gear icon, the top of the settings pane will display "WebContainers" for WebContainer-based projects or "Dev Server" for EngineBlock projects.
```
Comment (kc0tlh) [inline docs/guides/user-guide/available-environments.md:71]: Only true in the Classic Editor

Comment (kc0tlh) [inline docs/guides/user-guide/available-environments.md:71]: Only true for the Classic Editor

File: docs/guides/user-guide/available-environments.md [hunk 2]
```diff
@@ -56,3 +56,22 @@ This environment has a terminal that supports various common commands you would
 WebContainers are currently supported by Chromium-based browsers and [in Firefox, with some caveats](/platform/webcontainers/browser-support).
 
 Our [Codeflow IDE](/codeflow/what-is-codeflow) runs on the WebContainers runtime.
+
+# Determining Which Environment You Are Using
+
+:::tip Not seeing a terminal in your project?
+
+Currently, the terminal is only available in projects which use the WebContainers execution environment. There is no way to automatically convert from one execution environment to another.
```
Comment (kc0tlh) [inline docs/guides/user-guide/available-environments.md:64]: Don't tell them they can't autop convert without providing the workaround. Probably best ot just not mention it at all

**Review by kc0tlh — CHANGES_REQUESTED** [review]
See comments

**Comment by purplem1lk** [comment]
@kc0tlh feedback has been incorporated. @sulco PR is ready for you when you are! 🙏

File: docs/guides/user-guide/available-environments.md [hunk 3]
```diff
@@ -56,3 +56,19 @@ This environment has a terminal that supports various common commands you would
...
 
 Our [Codeflow IDE](/codeflow/what-is-codeflow) runs on the WebContainers runtime.
+
+# Determining Which Environment You Are Using
+
+The simplest way to determine if you're using EngineBlock vs WebContainers is the presence of a terminal, as shown below. Currently, the terminal is only available in projects which use the WebContainers execution environment. 
+
+![Screenshot of the terminal in a StackBlitz WebContainer project](./assets/stackblitz_terminal.png)
+
+You can also check in the project settings in the left pane of any StackBlitz project. When you click the gear icon, the top of the settings pane will display "WebContainers" for WebContainer-based projects or "Dev Server" for EngineBlock projects.
```
Comment (purplem1lk) [inline docs/guides/user-guide/available-environments.md:66]: Also add Codeflow behavior and screenshots

**Comment by purplem1lk** [comment]
@kc0tlh ⬆️ Pinging for review to enable merging.

**Review by kc0tlh — APPROVED** [review]

**Comment by peter-zawistowicz** [comment]
@kc0tlh this looks great! the one question I had was whether the title is explicit enough. maybe something like: "Runtimes: EngineBlock vs. WebContainers" ?

**Comment by kc0tlh** [comment]
Re-created and merge this branch in PR #160

**Pull request merged**